src/builder/copy.ts

```diff
@@ -0,0 +1,52 @@
+import { relative, resolve } from "pathe";
+import { globby } from "globby";
+import { copy } from "fs-extra";
```

Comment:
We shall use native [`fs.copyFile`](https://nodejs.org/api/fs.html#fspromisescopyfilesrc-dest-mode) introduced in Node 14.

I think we can also opt-in or try `fs.cp` when available even if experimental, it can give better performances.

Reply:
@pi0 you're right! On the first iteration, I was thinking of allowing folder-to-folder copy which is not supported by `fs.copyFile` (I didn't know about `fs.cp`), but in the end, I opted for file-to-file only and forgot to remove `fs-extra.` Fixed in https://github.com/unjs/unbuild/pull/389/commits/034b89a50fe6268e5198b91b8c86a5465b1fc358. 

By the way, in https://github.com/unjs/unbuild/pull/389/commits/80529babb7c698c13d1cc5b0f6a649002305781a I update the builder to use `fs.cp`, if available, with a simple:

```ts
const copy = fsp.cp || fsp.copyFile;
```

I guess that should be enough for systems not supporting `fs.cp` or should I add additional checks?